app/helpers/hyrax/hyrax_helper_behavior.rb

```diff
@@ -54,6 +54,23 @@ def browser_supports_directory_upload?
       user_agent.include? 'Chrome'
     end
 
+    # @param [User] user
```

Comment:
I replaced the notify_number partial with this helper, since there's a fair bit of logic here.